### PR TITLE
BlogList

### PR DESCRIPTION
**Pull request by teaskvarc**
Made grid for blog list. Added new UI component Typography.

### CODE REVIEW

**Comment by teaskvarc** [comment]
TO DO:
1. Add exposed / first blog (the big one)
2. BlogListItem need data: tag, content, author (now is static).
3. Correct date format

File: components/UiKit/LinkAsButton.tsx [hunk 1]
```diff
@@ -28,13 +28,14 @@ const LinkAsButton = styled.a`
...
         background-image: ${({ theme }) =>
-            `linear-gradient(180deg, ${theme.orangeHover} 100%, ${theme.orangeGradientColor1} 100%)`};
+        `linear-gradient(180deg, ${theme.orangeHover} 100%, ${theme.orangeGradientColor1} 100%)`};
         opacity: 0;
         z-index: -100;
         transition: opacity 0.45s;
     }
 
     &:hover {
+        color: ${({ theme }) => theme.white};
```
Comment (mitjapotocin) [inline components/UiKit/LinkAsButton.tsx:38]: I don't mind, if you use just #fff for white, but its your choice

File: components/UiKit/Typography.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export const Heading3 = styled.h3`
+    font-size: 1.375rem;
```
Comment (mitjapotocin) [inline components/UiKit/Typography.tsx:4]: Lets use pixels for font-size everywhere, its easier to use figma numbers, and we don't need a calculator.

File: pages/blog/index.tsx [hunk 3]
```diff
@@ -1,23 +1,70 @@
...
+    aspect-ratio: 1.79;
+    object-fit: cover;
+    width: 100%;
+  }
+`;
+
+const BlogTag =  styled.p`
+  font-size: 1rem;
+  line-height: 1.25;
+  font-weight: ${({theme})=> theme.fontWeight600};
```
Comment (mitjapotocin) [inline pages/blog/index.tsx:65]: Isn't it easier to just write `font-weight: 600`?

I don't see any benefit of havingh
```
  fontWeight400: 400,
  fontWeight600: 600,
  fontWright700: 700,
```
as a variable

**Pull request merged**